recipe/meta.yaml

```diff
@@ -64,7 +74,7 @@ about:
     information systems with other computer systems.
```

Comment:
Could you add this license https://github.com/Toblerity/Fiona/blob/master/fiona/_vendor/munch/LICENSE.txt as `munch` is vendored starting from v1.9.4, see https://github.com/Toblerity/Fiona/blob/195579d8737afdd99dfab6db8315815fa12d0f18/CHANGES.txt#L42

Reply:
Umm, good catch.

Reply:
thank you, addressed.